### ISSUE TITLE
Support for Heroku 24 stack

### ISSUE THREAD

**Issue by everton** [RohanDebroy/heroku-buildpack-wkhtmltopdf#5]
Heroku 24 was released, and it would be nice to have this buildpack running on it.

**Comment by jeremygradisher** [comment]
Agreed. I had just switched to this to get away from wkhtmltopdf-binary which was adding like 300mb to my slug size. I would love to see this for Heroku 24.
